### DISCUSSION TITLE
Moneybuster, cospend and Ihatemoney : should we unite under a same identity?

### DISCUSSION THREAD

**Discussion by almet** [spiral-project/ihatemoney#1198] [General]
We've already discussed that in the past : we have different applications which interact together, but with different names. This seem to complicate things for the end users, as they have to remember different names depending the platform they are using.

On android, you use Money Buster, if you're hosted on Nextcloud you use cospend, and if you host things yourself, or depend on the hosted version, then you're using IHateMoney.

This situation could be better, but that would require some coordination between the different projects. I think it's worth it, but also I'm not sure under which name we should federate. I'm thinking about : 

- Making a website which makes it clear what we do, and propose to download and install the different applications ;
- Closing the gap between the different servers. I'm not sure we want to integrate all of the features from cospend, but some of them might be useful, or at least we should document them ;
- Maybe should we try to integrate better between these tools. We currently have import/export features which are potentially broken, for instance.

What do you think about this idea? @Glandos, @zorun @julien-nc ?

And also : do you have an idea for a name? =D

**Comment by adan-ea** [comment]
Hi there, this might be old and i might be out of bound but I do agree with you. 
The fact that you have multiple apps is a bit confusing.

First time I tried IHateMoney, I saw on the web page that there was an app so i took my phone and looked for IHateMoney on F-Droid (since that's the name I know) but didnt find it. Then went to Aurora and got showed MoneyBuster by looking for "IHateMoney" and didn't bother to look at it because it wasn't the same name. 
Until today where I learned about MoneyBuster with this post (entirely on me since I didn't do my proper research) 

But the project would be better off under the same identity. I do prefer cospend for the name between the three provided :]
